test/Altinn.Platform.Authentication.Tests/RepositoryDataAccess/DbTestBase.cs

```diff
@@ -64,7 +64,7 @@ async Task IAsyncLifetime.InitializeAsync()
         container.Configure<PostgreSqlSettings>(settings =>
         {
             settings.ConnectionString = connectionString;
-            settings.AuthenticationDbPwd = "unused";
+            settings.AuthenticationDbPassword = "unused";
```

Comment:
Isnt it more readable to have these information on test configuration file?

Reply:
yes, but they are not used, the package generates internal connectionstring automatically